### DISCUSSION TITLE
Using `nvim` for opening downloads

### DISCUSSION THREAD

**Discussion by pinarruiz** [qutebrowser/qutebrowser#8186] [Q&A] [answered]
I have setted my `xdg-open` to use `nvim.desktop`, but using the CTRL+x to open download on a `json` file for example brings nothing up. Using it on a terminal, reuses the terminal, and opening qutebrowser from a terminal also reuses that terminal, is there anything I could do?

Thanks.

**Comment by The-Compiler** [comment, marked as answer]
Nothing specific to qutebrowser here. `xdg-open` [ignores](https://gitlab.freedesktop.org/xdg/xdg-utils/-/issues/84) `Terminal=true`, so you'll need a `.desktop` file that opens vim in a terminal applicaton of your choice. See e.g. [arch linux - Set default xdg-open application to terminal program - Unix & Linux Stack Exchange](https://unix.stackexchange.com/questions/177976/set-default-xdg-open-application-to-terminal-program) which discusses this at length.

Alternatively, use a GUI application, e.g. `nvim-qt`.
